css/styles.css

```
@@ -3,6 +3,7 @@
 /*Components*/
 @import "components/chaemin/nav-bar.css";
 @import "components/chaemin/width.css";
+@import "components/sangchun/nav-top.css";
 
 /*Screens*/
 
```

```suggestion
@import "screens/sangchun/mypage.css";
```
css 파일은 새로 만들 때마다 styles.css 에 import 해주고,
mypage.html 에서 link한 mypage.css는 지우면 좋을 것 같습니당!
(html 파일에 링크하는 css 파일은 styles.css 하나만)